src/components/data_validation_overlay/dv_list_icon/dv_list_icon.scss

```diff
@@ -0,0 +1,5 @@
+.o-spreadsheet {
```

Comment:
css won't work with my PR to put all the icons in the canvas. Actually we should discuss how both task will interact, because I don't think it'll be easy